app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -174,5 +205,10 @@ def valid_claim_data(saved_claim_id, vet_info)
 
       claim.formatted_686_data(vet_info)
     end
+
+    def split_file_and_path
+      path_split = path.split('/')
+      { file: path_split[...-1].join('/'), file_name: path_split.last }
```

Comment:
I think the intention with this when I was doing it is that file is the full path to the file including file name, and file name was the original name of the file. 

This was done this way because the generation would often result in random generated doc names so for example it would be like 

```
{ file: "tmp/04a0b668-67d5-11ee-8c99-0242ac120002.pdf", file_name: "jeffs_evidence_doctors_note.pdf" }
```
If that is not happening here, and you are controlling your filenames fully in this process (as opposed to me who was relying on existing code for a lot of components of the solution) you may not need to do this but I would just need to verify that the function works properly in both cases. Like in my case I use just the `file` key to access the actual file on the filesystem, this looks like you would need to use both to get the actual location on the filesystem, so... just need to make sure it supports that....

Reply:
Under further investigation, this could be the reason for this failure maybe? https://github.com/department-of-veterans-affairs/vets-api/runs/17582675826

It is erroring a test saying it cannot find the file, looks like because it has an incomplete path to the file, which could be a side effect of what I am bringing up here (or could be unrelated, idk)

Just something to look into maybe.